### ISSUE TITLE
Auto tagget file are not deleted in External Storage folders

### ISSUE THREAD

**Issue by GAS85** [nextcloud/files_retention#79]
### Steps to reproduce
1. Create TAG with retention rule 
![image](https://user-images.githubusercontent.com/6813635/70983838-b3743f00-20b9-11ea-9fd1-8d312fc8ef0b.png)

2. Set tag on a file on a external storage 
![image](https://user-images.githubusercontent.com/6813635/70984013-02ba6f80-20ba-11ea-9b0d-559afb8e70f5.png)

3. See that it will not be deleted when date appears. Today is already 62 Days: 
![image](https://user-images.githubusercontent.com/6813635/70985186-06e78c80-20bc-11ea-8b5c-5627fd735025.png)

### Expected behaviour
File deleted when tag is set and final date was reached

### Actual behaviour
No files deleted on external storages

### Server configuration

**Operating system**: Ubuntu 18.04

**Web server:** Apache/2.4.41

**Database:** 10.1.43-MariaDB

**PHP version:** PHP-fpm 7.3.12

**Nextcloud version:**  16.0.6

**Where did you install Nextcloud from:** Official

**List of activated apps:** 

```
Enabled:
  - accessibility: 1.2.0
  - activity: 2.9.1
  - admin_audit: 1.6.0
  - audioplayer: 2.8.4
  - bruteforcesettings: 1.4.0
  - calendar: 1.7.1
  - checksum: 0.4.3
  - cloud_federation_api: 0.2.0
  - comments: 1.6.0
  - data_request: 1.3.0
  - dav: 1.9.2
  - deck: 0.6.6
  - federatedfilesharing: 1.6.0
  - federation: 1.6.0
  - files: 1.11.0
  - files_automatedtagging: 1.6.0
  - files_external: 1.7.0
  - files_pdfviewer: 1.5.0
  - files_retention: 1.5.1
  - files_rightclick: 0.15.1
  - files_sharing: 1.8.0
  - files_texteditor: 2.8.0
  - files_trashbin: 1.6.0
  - files_versions: 1.9.0
  - files_videoplayer: 1.5.0
  - firstrunwizard: 2.5.0
  - flowupload: 0.1.7
  - gallery: 18.3.0
  - gpxpod: 4.1.0
  - keeweb: 0.5.1
  - logreader: 2.1.0
  - lookup_server_connector: 1.4.0
  - mail: 0.17.0
  - maps: 0.1.2
  - nextcloud_announcements: 1.5.0
  - notes: 3.0.3
  - notifications: 2.4.1
  - oauth2: 1.4.2
  - password_policy: 1.6.0
  - phonetrack: 0.5.2
  - polls: 0.10.4
  - previewgenerator: 2.2.0
  - privacy: 1.0.0
  - provisioning_api: 1.6.0
  - radio: 0.6.5
  - recommendations: 0.4.0
  - serverinfo: 1.6.0
  - sharebymail: 1.6.0
  - survey_client: 1.4.0
  - systemtags: 1.6.0
  - theming: 1.7.0
  - twofactor_backupcodes: 1.5.0
  - twofactor_totp: 3.0.1
  - unsplash: 1.1.4
  - updatenotification: 1.6.0
  - viewer: 1.2.0
  - weather: 1.6.4
  - workflowengine: 1.6.0
Disabled:
  - encryption
  - impersonate
  - sharerenamer
  - support
  - user_ldap
  - workflow_script
```

**Nextcloud configuration:**

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": {
            "0": "***REMOVED SENSITIVE VALUE***",
            "2": "***REMOVED SENSITIVE VALUE***"
        },
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/gas85.ddns.net\/nextcloud",
        "dbtype": "mysql",
        "version": "16.0.6.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "filesystem_check_changes": 0,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "timeout": 1.5
        },
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "theme": "",
        "logfile": "\/var\/nextcloud\/data\/nextcloud.log",
        "loglevel": 1,
        "trashbin_retention_obligation": "14, auto",
        "versions_retention_obligation": "14, 180",
        "data-fingerprint": "***REMOVED SENSITIVE VALUE***",
        "enable_previews": true,
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\BMP",
            "OC\\Preview\\XBitmap",
            "OC\\Preview\\Movie",
            "OC\\Preview\\PDF",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\MarkDown"
        ],
        "preview_max_x": 1080,
        "preview_max_y": 1920,
        "auth.bruteforce.protection.enabled": true,
        "simpleSignUpLink.shown": false,
        "mail_smtpsecure": "tls",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "twofactor_enforced": "true",
        "twofactor_enforced_groups": [
            "admin"
        ],
        "twofactor_enforced_excluded_groups": [],
        "has_rebuilt_cache": true
    }
}
```

**Comment by GAS85** [comment]
Files are still on the place:
![image](https://user-images.githubusercontent.com/6813635/72536178-0f2d1600-387a-11ea-8d67-e16cd19cbb9c.png)

Also Local files tagged are not deleted automatically. What I did wrong?

**Comment by GAS85** [comment]
Seems works in NC 17+

**Comment by sualko** [comment]
Should be fixed with c456ed991aa12076571f9355236e0cf620a60cf6. Probably the filesystem was not loaded. Please reopen the issue if your problem persist.
